arangod/ClusterEngine/ClusterIndex.cpp

```diff
@@ -169,66 +169,6 @@ void ClusterIndex::toVelocyPack(
     builder.close();
   }
```

Comment:
We probably can remove a few header includes here, too.

Reply:
AgencyCache.h, LogMacros.h, Utilites.h, Network/Methods.h, Network/NetworkFeature.h I would expect from what we have seen in 3.11.